scripts/motdgen

```diff
@@ -11,7 +11,7 @@
 source /usr/lib/os-release
 
 mkdir -p /run/coreos
-echo -e "Core\033[38;5;206mO\033[38;5;45mS\033[39m ${GROUP} (${VERSION})" > /run/coreos/motd
+echo -e "\e[${ANSI_COLOR}m${NAME}\e[39m ${GROUP^} (${VERSION})" > /run/coreos/motd
```

Comment:
Group might be a UUID or other identifier in custom CoreUpdate setups. Just leave the case alone.